src/components/Modals/MyPlaceDeleteModal/MyPlaceDeleteModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Button, Modal } from "@mui/material";
+import styles from "./MyPlaceDeleteModal.module.css";
+
+type Props = {
+    openModal: boolean;
+    closeHandler: () => void;
+    confirmDelete: (id: string) => void;
+    id: string;
+  };
+
+const MyPlaceDeleteModal = (props: Props) => {
+    const style = {
+        position: "absolute" as "absolute",
```

Comment:
If you had problems with types here try setting the type of the style variable to `CSSProperties`